src/main/java/com/synopsys/arc/jenkinsci/plugins/extratoolinstallers/utils/ExtraToolInstallersException.java

```diff
@@ -31,6 +31,7 @@
  * @author Oleg Nenashev
  */
 public class ExtraToolInstallersException extends IOException {
+    private static final long serialVersionUID = -6131475303804497293L;
```

Comment:
It is better to be careful with introducing serial version IDs for old classes. But it should work fine in this case